eng/Version.Details.xml

```diff
@@ -330,6 +332,10 @@
       <Uri>https://github.com/dotnet/command-line-api</Uri>
       <Sha>a045dd54a4c44723c215d992288160eb1401bb7f</Sha>
     </Dependency>
+    <Dependency Name="System.CommandLine.Rendering" Version="0.4.0-alpha.23407.1">
```

Comment:
The Rendering version is very different than the above S.CL version. Is format way behind on S.CL? That'll likely need to be sorted out.

Reply:
no I think the rendering package just doesn't see many updates - https://www.nuget.org/packages/System.CommandLine.Rendering/#versions-body-tab